homeworks/12-mars-mobx/src/store.tsx

```diff
@@ -0,0 +1,86 @@
+import { action, computed, makeAutoObservable } from "mobx";
+import { PhotoJson, Photo } from "types/Photo";
+import { createContext } from "./storeUtils";
+
+const MARS_API_KEY = "aZ6TQDnvVnCzV1OsLt3ExZDIa0CpV38lvafz9dZi";
+
+class Sols {
+  selectedSol: string = "1";
+  isLoading: boolean = false;
+  sols: {
+    [key: string]: Photo[];
+  } = {};
+
+  constructor() {
+    makeAutoObservable(this, {
+      setSol: action.bound,
+      loadSolPhotos: action.bound,
+      currentSolPhotos: computed,
+    });
+  }
+
+  get currentSolPhotos(): Photo[] | undefined {
```

Comment:
If you are using a getter then the properties return should be private.

With `sols` being public (if no access modifier defined then it is automatically public) I can just do `this.sols[this.selectedSol]` without the getter.